packages/components/src/particles/Banner/index.tsx

```diff
@@ -1,21 +1,46 @@
-import { Show, JSXElement } from 'solid-js';
+import { createSignal, JSXElement, onMount, Show } from 'solid-js';
 import Icon from '../Icon';
 
 export type BannerProps = {
   children: JSXElement;
   status: 'success' | 'info';
   withoutIcon?: boolean;
+  closable?: boolean;
+  name?: string;
```

Comment:
not a defect just a thought - seems like this is more of an `identifier` than a name? feel free to disregard i can just see the word `name` obscuring the true purpose of this prop

Reply:
Agree with Bill

Reply:
That makes sense, updated!